README.md

```diff
@@ -1,4 +1,6 @@
 # Go_structor [![Actions Status](https://github.com/goreflect/go_structor/workflows/CI_dev/badge.svg)](https://github.com/goreflect/go_structor/actions?query=workflow%3ACI_dev) [![Go Report Card](https://goreportcard.com/badge/github.com/goreflect/go_structor)](https://goreportcard.com/report/github.com/goreflect/go_structor)
```

Comment:
Change repo name everywhere if it's been renamed

Reply:
++

Reply:
Some more places
![image](https://user-images.githubusercontent.com/231479/72679527-97155900-3ae2-11ea-82f5-658810d55031.png)

Reply:
done